test/unit/image_s3_test.rb

```diff
@@ -12,9 +12,7 @@ def test_basic_stuff
       stub: true # (makes Aws::S3 return empty responses for everything)
     )
     assert(s3.status("key"))
-    s3.list.each do |_obj|
-      break
-    end
+    assert(s3.list.methods.include?(:each))
```

Comment:
Was this the intent of the old code?
Or was it intended to see if an exception is raised when `s3.list` is called. If so, then something like
```ruby
assert_nothing_raised do
  assert(s3.list.methods.include?(:each))
end
```

Reply:
What happened to my earlier comment?  Darned flaky internet...  The point here is to make sure Results#each works or at least doesn't crash or something.  If rubocop requires something in the body of the block, then throw in an "assert(_obj)".  That should keep it happy, right?

Reply:
Thanks for educating me. I'll fix this when I'm more awake. 
(I hope I didn't accidentally delete your earlier comment, but I don't recall seeing it.)

Reply:
I simply disabled the relevant cop around these lines.